package.json

```diff
@@ -1,28 +1,41 @@
 {
   "name": "pidusage",
-  "version": "2.0.3",
-  "description": "Cross-platform process cpu % and memory usage of a PID — Edit",
+  "version": "2.0.0",
+  "description": "Cross-platform process cpu % and memory usage of a PID",
+  "license": "MIT",
+  "homepage": "https://github.com/soyuka/pidusage",
+  "repository": "github:soyuka/pidusage",
+  "bugs": {
+    "url": "https://github.com/soyuka/pidusage/issues"
+  },
+  "author": "soyuka",
+  "contributors": [
+    "Simone Primarosa <simonepri@outlook.com> (https://simoneprimarosa.com)"
+  ],
   "main": "index.js",
+  "files": [
+    "lib",
+    "index.js"
+  ],
+  "engines": {
+    "node": ">=4"
+  },
+  "scripts": {
+    "lint": "standard index.js lib/*.js test/*.js",
+    "test": "npm run lint&& nyc ava -m \"!*benchmark*\"",
+    "bench": "ava -m \"*benchmark*\""
+  },
   "dependencies": {},
   "devDependencies": {
-    "faucet": "0.0.1",
+    "ava": "*",
+    "mockdate": "^2.0.2",
     "mockery": "1.4.0",
-    "nanobench": "^2.1.1",
-    "ps-tree-ce": "^1.1.0",
+    "nyc": "^11.6.0",
```

Comment:
what's this for?

Reply:
nyc is for the coverage report:
![image](https://user-images.githubusercontent.com/3505087/37570170-81052d82-2aec-11e8-95df-4a1539722ada.png)